.gitignore

```diff
@@ -269,3 +269,11 @@ binaries/
 
 # Do not include launch settings
 launchSettings.json
+
+.dotnet/
+artifacts/
+/.packages
+eng/common/cross/
+eng/common/native/
+eng/common/performance/
```

Comment:
Obviously people can add this in the future if they need to, but perhaps we should keep performance? Usually stuff that goes in corefxlab is for testing and see if it is something we want to move to runtime repo, so perhaps being able to run performance tests is important.

Reply:
I would leave it out at the moment. This are mostly yml files and scripts to run performance tests that live on the perf repo. I think we can just remove it whenever we enable those perf builds.